webdriver/client.py

```diff
@@ -385,12 +377,13 @@ def send_command(self, method, uri, body=None):
 
         url = urlparse.urljoin("session/%s/" % self.session_id, uri)
         response = self.send_raw_command(method, url, body)
+        value = response.body["value"]
 
-        rv = response.body["value"]
-        if not rv:
-            rv = None
+        if response.status != 200:
```

Comment:
On the third hand, this implicit behavior will tend to make the exceptions
raised in response to user errors a little more confusing. Attempting to send
"New Session" when one already exists, for instance, will produce an invalid
URL and result in an "Unknown Command" exception (rather than a "Session Not
Created" exception). Similarly, attempting to issue a session-dependent command
(e.g. "Get Current URL") without a valid session will produce an "Unknown
Command" exception rather than a "Session Not Created" exception.